extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -28,7 +28,8 @@ const buildFiles: string[] = [
 	'Microsoft.SqlServer.Types.dll',
 	'System.ComponentModel.Composition.dll',
 	'System.IO.Packaging.dll',
-	'Microsoft.Data.Tools.Schema.SqlTasks.targets'
+	'Microsoft.Data.Tools.Schema.SqlTasks.targets',
+	'Microsoft.SqlServer.Server'
```

Comment:
Microsoft.SqlServer.Server had to be added so it'll get copied over for build to work, like @zijchen mentioned [here](https://github.com/microsoft/DacFx/pull/164#discussion_r1004941069)